### DISCUSSION TITLE
Who's Hiring (Aug 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#53811] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #53810.

Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by VanessaCrid** [comment]
[Arcade Software](https://www.arcade.software/) | Full Stack Engineer | San Francisco, CA (hybrid 2x/week) | Full-time | $175k-$200k base, .1-.3% equity | [Apply Here!](https://jobs.ashbyhq.com/arcade/0d5c297e-fd18-4afc-9a7b-aaadc4bffe81)

Arcade is a seed-stage start up that builds software for product marketers - our first product is a demo-making tool. Our mission is to empower the Product Led Growth economy by making it super simple to create beautiful demos that tell the story of their product.

We are currently looking for Full Stack Product Engineers who embrace product-led development by understanding and empathizing with our user's challenges and business needs. You'll join a highly collaborative and fun team to build delightful products that value the user experience!

On the front end, our stack is heavily Jamstack-based with Next.js, Typescript, as well as a Chrome Extension. On the back end, we use GCP (Firestore, Cloud Functions, PubSub, IdP, and a few other services) and AWS (Aurora). Arcades are embedded on our users' site, which means we strive for high-performing code and we must maintain high performance to make that work well. We have some really interesting challenges around video processing and AI on our roadmap.

If you're excited about moving fast, love working directly with users/customers and want an amazing team culture, this is the perfect fit for you! Please feel free to apply directly or share your resume/portfolio to me at [vanessa@arcade.software](mailto:vanessa@arcade.software)!

**Reply by IsomerX** [reply]
Location : Dhruv Bakshi
Remote : Yes
Willing to relocate: Flexible
Website : dhruvbakshi.com
Email : [dhruvbakshi12343@gmail.com](mailto:dhruvbakshi12343@gmail.com)
Resume :[Resume](https://drive.google.com/file/d/1hwm87Q_W4gQgXKy4RmNprB5uRWmcBzS6/view?usp=drive_link)
Linkedin : https://www.linkedin.com/in/dhruvbakshiwork/

Hey, I am Dhruv Bakshi, a fullstack developer with 4 years of experience. I would love to join your team for a fulltime/part time position.

Thank you

Dhruv Bakshi

**Reply by tawseefnabi** [reply]
hi @VanessaCrid 
Thanks for giving me an opportunity to apply your company

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Comment by leerob** [comment]
> We are hiring 10 Next.JS full stack developers to build some really cool products with 
[@WhopIO](https://twitter.com/WhopIO). Please message me a product you have built and what your favorite app is if you are available for work.

https://twitter.com/cultured/status/1690917676200206336

**Reply by SuperButterfly** [reply]
Hi leerob
Nice to meet you. I am really happy to contact you.
I will give you my previous works
https://vergecurrency.com/
Verge is a digital currency that allows for blockchain payments.

https://nubank.com.br/en/
Nubank offers a digital account similar to Revolut for Brazilians.

Please check these sites and contact me.

Thanks

**Reply by Favourz1** [reply]
Hello @leerob , Apparently there's no way to reach you via dm's

A project i have built amongst others: www.favourz-codepen.netlify.app

My resume: https://drive.google.com/file/d/15QktDjk6KiwXN4CaRNgGfpHPhw19bXR0/view?usp=drivesdk

Email: okohfavour91@gmail.com

**Reply by vegawong** [reply]
Hello @leerob ， is this a remote job?

**Reply by tawseefnabi** [reply]
Hi @leerob 
Thanks for giving me an opportunity to apply your project.

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Reply by apophis51** [reply]
hello I'm updating my portfolio with https://profile.malcmind.com which is next.js based. I have a ton of projects listed on My Projects page.  My ICSP crypto was a small project but probably the coolest as well as [flixquest](https://flixquest-production.up.railway.app/).

**Comment by Marcfeitosa** [comment]
ARKHO está buscando un Ingeniero Cloud.

💰 Compensación: USD2.3k - 2.7k/mes aceptables.

+ Path de entrenamiento
+ Path de entrenamiento
+ Path de entrenamiento
🌎 Ubicación: Remote

💪 Habilidades:

Eres competente en Computación en la nube
Eres competente en Java
Eres competente en JavaScript
Eres competente en TypeScript
Eres principiante en AWS
Eres principiante en nginx
Estás interesado en REST API


Aplicar a través de Torre es súper fácil usando el siguiente link: https://torre.ai/post/Nwqmnolw



⭐ Responsabilidades:

Estamos en busca de un profesional con experiencia en desarrollo, arquitectura de software, aplicando conocimientos técnicos en la ejecución de proyectos tecnológicos.



La vacante considera que el aspirante sea un referente orientando a la mejora continua y el cumplimiento de los objetivos y la creación de productos tecnológicos de alta calidad.



Para ello, la experiencia en el uso de diversas herramientas, lenguajes de programación, infraestructura y otros, son la base para apoyar desde el conocimiento al equipo de desarrollo, participando en el desarrollo de entregables de alto impacto y calidad.



🟠 Sus responsabilidades incluirán:

⚡Trabajar de forma integrada y colaborativa con el líder de equipo, y el equipo de implementación proporcionando un contexto claro del estado de las iniciativas desde el ámbito tecnológico.

⚡Apoyar al equipo en la definición, estimación y planificación de tareas/actividades para el desarrollo del producto en el marco ágil del desarrollo.

Involucramiento en el desarrollo de componentes más complejos y críticos de los sistemas.

⚡Diseñar e implementar arquitecturas de referencia para la implementación.

Profundización en aspectos tecnológicos desconocidos y que se requieren para el logro de los objetivos.



🟠 Información adicional:

⚡ Nos gusta construir equipos colaborativos y autogestionados, por lo que tener buenas habilidades de comunicación y responsabilidad es muy importante.

⚡ Encontrará desafíos con las tecnologías en la nube que requerirán esfuerzo y certificación.

⚡Buscamos excelentes profesionales que puedan jugar dentro y fuera del campo.



🟠 Perfil del rol:

⚡Experiencia mínima de 2 años en Python, Spark, DevOps. Despliegues AWS: CDK desarrollando en equipos TI y en arquitecturas modernas.

⚡Conocimiento de arquitectura de software, patrones de diseño y experiencia en el liderazgo de construcción de productos digitales.

⚡Stack Desarrollo: NodeJS, Python, JavaScript, PHP, Java, React, etc.

⚡Stack WebServer: Apache, Tomcat, NGINX, etc

⚡Storage Technologies (SAN/NAS)

⚡APIs: Consumo, construcción e integración; API Gateway, etc.

⚡Conocimiento de micro-servicios y contenedores.

⚡Experiencia comprobable bajo el marco de mejora continua y CI/CD.

⚡Conocimientos de herramientas de gestión ágil.



🟠 Es una ventaja si tiene experiencia con algunas de las siguientes tecnologías:

⚡ DevOps: Infra como codificación, CDK, Pipelines.

⚡AWS: Glue, Lambda, Redshift, Codedeploy, Codebuild (Entender y desarrollar en arquitecturas Serverless). Saber desarrollar en Cloud.


Aclaración: Las aplicaciones recibidas en esta publicación serán procesadas vía torre.ai.

**Reply by SuperButterfly** [reply]
Hi. I am really interested in your position
[Uploading 1.pdf…]()

Please contact me 
Thanks

**Reply by Marcfeitosa** [reply]
Hey there, you have to apply using the link in the description. The hiring team for this position will contact you.
You only must be located in LatAm region

**Reply by tawseefnabi** [reply]
Hi Dene 
Thanks for giving me an opportunity to apply your project.

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Comment by klink** [comment]
Refact.ai | https://refact.ai/ | Developer Advocate | Software Engineer | Remote (UK & Europe) | 80 - 120K + equity

We're building the best open-source AI for coding using all the best LLMs available on the internet (GPT, Starcoder, WizardCoder, LLama2).

We’re hiring for Refact’s first Developer Advocate. As the first Developer Advocate, you'll play a key role in helping developers use the power of AI for coding. You'll have the opportunity to grow and support a community of developers who are excited about the future of AI and open-source.

We're looking for a Software Engineer to help us build the best developer experience for the AI code assistants inside IDEs. We currently have VS Code and JetBrains plugins and plan to add more IDEs and more functionality to the plugins.

Apply:

Developer Advocate https://wellfound.com/l/2yXwiJ
Software Engineer: https://wellfound.com/l/2zdYHM

**Reply by tawseefnabi** [reply]
hi @klink 
Thanks for giving me an opportunity to apply your company

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Comment by vdeez1** [comment]
Ethica | Product Engineer | On-Site (Mumbai, India) - Open to remote | Full-time
https://ethica-ai.notion.site/Product-Engineer-ethica-76d26ea5727d41d6885f0c71fb57023a

Ethica is hiring engineers located in Mumbai to build out our India engineering team. The position is on-site, but we may be open to remote work for qualified candidates. This is an exciting time to join the company, we are growing quickly! Please apply through the above link

**Reply by codepool867** [reply]
Hi @vdeez1 I am pretty much interested in your offer.
This is Joris from France, a senior fullstack engineer with 7 years of professional experience.
If it is allowed, I would love to work remotely.
I already applied with the link you provided, but followings are some of my information for your reference.
Email: jorisbaly1@gmail.com
Resume: https://drive.google.com/file/d/1XKO_fLQpqoY_7X8GGuQ5SeQC689LS35T/view?usp=sharing
Linkedin: https://linkedin.com/in/joris-baly
Githug: https://github.com/devofficer
Looking forward to hearing from you soon.
Warm regards,
Joris

**Comment by dhruv92** [comment]
[Create](https://create.xyz) | Freelance Full Stack Web Developer | Remote (US timezone overlap) | Contract | [Apply here](https://jobs.ashbyhq.com/create/4cbc0719-91ad-44c8-9212-fe0880b6906a)

Create helps founders build web applications lighting fast using our generative AI powered code generation system and an on demand, vetted community of top web developers. We work with VC backed startups and build real production products that matter to users...in a fraction of the typical time. We're rapidly bringing on more projects, so also growing the developer community. We're a team of ~14 developers, and we're looking for more experts in Typescript, React/Next, Node, GraphQL, REST, Postgres, etc. to join the team. 

Some benefits of freelancing with us include: fast learning, code generation that automates the common parts of product creation, large project variety, an active community of experienced developers to collaborate with, flexible hours, and top rates based on experience level and geo.

**Reply by tawseefnabi** [reply]
hi @dhruv92 
Thanks for giving me an opportunity to apply your company

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Reply by dantaeil** [reply]
Hello!
I'm a Full Stack Developer with 8+ years of experience with a solid foundation of React, Next, Vue, Nuxt.
I also have experience in Backend side like Express, Node.js, MongoDB, MySQL.

Resume: [Dan_Johnson_Resume.pdf](https://github.com/vercel/next.js/files/12516878/Dan_Johnson_Resume.pdf)
Email: [danjohnson0037@gmail.com](mailto:danjohnson0037@gmail.com)

In past positions, I have been a really good collaborator and contributor to the team.
And I am also eager to learn latest technologies and stay up-to-dating.
I want to contribute to your team.

**Comment by mustafadalga** [comment]
**Location:** Turkey
**Remote:** Yes
**Willing to relocate:** Yes
**Technologies:** React.js, Vue.js, TypeScript, JavaScript, Next.js, Vuex, Redux Toolkit, Zustand, Webpack, Vite.js, Tailwind CSS, SCSS, CSS3, HTML5, Pinia.
**Resume/CV:** https://mustafadalga.github.io/cv -  https://www.linkedin.com/in/mustafadalga/
**Website:** https://github.com/mustafadalga/ - https://sft.hashnode.dev/
**Email:** mustafadalgaa <at> gmail <dot> com

Hi 👋,
 I am skilled Front-End Developer based in Turkey with 5 years of experience. I am looking a new opportunity.

**Reply by kamaalsultan** [reply]
Maybe you wanna move your post to https://github.com/vercel/next.js/discussions/53810

**Reply by mustafadalga** [reply]
I have moved. Thank you.

**Comment by a-libre** [comment]
[Create](https://create.xyz/) | Freelance Full Stack Web Developer | Remote (US timezone overlap) | Contract | [Apply here](https://jobs.ashbyhq.com/create/4cbc0719-91ad-44c8-9212-fe0880b6906a)

Create helps founders build web applications lightning-fast using our AI-powered code generation system and an on-demand, vetted community of top web developers. We work with top VC-backed startups and build high-impact products that matter to millions of users… in a fraction of the typical time. We're rapidly bringing on more customers and projects, so we're also growing our freelance developer community. We're a team of ~14 developers currently, and we're looking for more experts in Typescript, React/Next, Node, GraphQL, REST, Postgres, etc. to join the team. Mostly looking for devs with 3+ years of experience, and excellent English-language communication skills so that our devs can be client-facing.

Some benefits of freelancing with us include: fast learning, code generation that automates the common parts of product creation, large project variety, an active community of experienced developers to collaborate with, flexible hours, and top rates based on experience level and geo.

Learn more here: https://gamma.app/docs/Become-a-Remote-FullStack-Developer-with-Create-Inc-u0fqnyj48sa2cro?mode=doc

**Reply by tawseefnabi** [reply]
hi @a-libre 
Thanks for giving me an opportunity to apply your company

Technologies: JavaScript, Node.js,React.js, Next.js, Python, Golang, Typescript, MySQL, MongoDB
[Resume](https://drive.google.com/file/d/10FZVrZFe1-1WlvBRQdlxQGk0uHPXUUGF/view?usp=drive_link)/CV Resume
Email: [tawseefnabi9@gmail.com](mailto:tawseefnabi@gmail.com)

I’m Tawseef Ahmad, a Full stack developer. I have around 9 years of experience in software development. I am skilled in JavaScript, Node.Js, Python,Golang and have extensive experience with modern JavaScript frameworks like ReactJs

**Reply by dantaeil** [reply]
Hello!
I'm a Full Stack Developer with 8+ years of experience with a solid foundation of React, Next, Vue, Nuxt.
I also have experience in Backend side like Express, Node.js, MongoDB, MySQL.

Resume: [Dan_Johnson_Resume.pdf](https://github.com/vercel/next.js/files/12516878/Dan_Johnson_Resume.pdf)
Email: danjohnson0037@gmail.com

In past positions, I have been a really good collaborator and contributor to the team.
And I am also eager to learn latest technologies and stay up-to-dating.
I want to contribute to your team.

**Comment by a-libre** [comment]
https://calendly.com/alex-create/non-technical-creator-interview

*Alex Libre*

talent @ create ( https://www.create.xyz/how-it-works ).
build ( https://app.create.xyz/gamma ) an app *|* join ( https://www.craft.me/s/Ad59ljNyvHOPs3 ) the team *|* become ( https://gamma.app/docs/Become-a-Creator-Create-Inc-u0fqnyj48sa2cro?mode=doc ) a creator

**Comment by Kshubhranshu** [comment]
Next.js | Frontend Developer | Internship | Remote (IST Timezone) | $120 per month | [Job Description Link](https://cuvette.tech/app/public/internship/64cd1e6d9c76606e9bae5b23) | [Company Link](https://gofibo.in/) | [Apply here](https://forms.gle/HAbq5YfiS2F91pSK6)

We at Fibo https://gofibo.in/ are looking Frontend Developer Intern. Fibo is SaaS platform to help you digitize and grow your fitness business with your own personalized mobile app. 

Please go through the job description: https://cuvette.tech/app/public/internship/64cd1e6d9c76606e9bae5b23

If you're interested for the role, you can apply directly on the link: https://forms.gle/HAbq5YfiS2F91pSK6

**Comment by Franetse7** [comment]
Hi admins just to thank you for the offers

Le lun. 4 sept. 2023 à 10:55, Tawseef Nabi ***@***.***> a
écrit :

**Comment by yassine-dw** [comment]
`North Inc (Formerly DeepWhale AI) | Front End Engineer | REMOTE | Monthly Contract | $3000-$4000/mo  | https://docs.google.com/document/d/1DAb2VwI38CZK2eeDelzXRpplOD8dFbTnpBnfVjeFcwo/edit?usp=sharing
`
Hey Y'all !
My name is Yassine Açoine, cofounder/cto of North^ and we are currently looking for a 🔥 Front End Engineer!

We are revamping our FinOps app and later on our landing page. We are looking for someone that will be able to quickly build on top of our vision of creating a modern app that will allow our customers to save massively on the cloud. You will work directly with me, have creative freedom, and will be challenged throughout the process (in a fun way).

If you are interested and think you can be a great fit, please share your best 2/3 latest web projects (I care more about what you have done/are able to do versus your background)

**Reply by alexkvazos** [reply]
Hello! Do you have an email I can reach out to? I have a very strong background on UI design and development.

[CV](https://drive.google.com/file/d/1cd0RjqPUAqbR9Y6G6eX4g2TA8xRq9kKZ/view?usp=drive_link)
[Example Work](https://drive.google.com/drive/folders/1mzEQcA87ZkF3xly3mTOJvGQNs8pLc3pM?usp=drive_link)
[Website](https://alexcavazos.com/)
[me@alexcavazos.com](mailto:me@alexcavazos.com)

**Reply by kamaalsultan** [reply]
Dear Yassine Açoine [@yassine-dw],
I am a senior front-end engineer with more than 7 years of experience in JavaScript SPA frameworks including React, Vue.js, Next.js and Nuxt.js and I am also familiar with back-end engineering.
Here are some of my past projects using Next.js or React.
https://gorewear.com
https://thegreattipoff.com
https://stuart.com/
As a front-end engineer, I have strong skills in API integration, state management and responsive design.
I also worked with some open source projects beyond front-end projects including:
https://ubiquity/ubiquibot
https://byteballet/html-objectifier
I am able to work independently and deliver high quality of product in tight deadline.
I always follow industry standard and always focus on code reusability.
I worked with more than 3 AI startups as a lead or senior developer and am familiar with fast-paced startups.
I love challenging environment and I am interested in working with startups, working with new brand and grow with it.
I am pretty much excited by DeepWhale AI's vision to creating a modern app that will allow our customers to save massively on the cloud.
I believe DeepWhale AI is the only place I would love to work on because its mission is a great match with my next-career goal.
Looking forward to your reply
Thanks very much and regards,
Shaorun

**Reply by kamaalsultan** [reply]
P.S. What makes me more interested and a more powerful candidate is my passion for AI and experience with AI startups.
You can reach out me at shaorun1010@gmail.com
Thanks

**Comment by codepool867** [comment]
Dear Yassine,

I hope this message finds you well. I wanted to express my appreciation for posting the job opportunity.

Allow me to introduce myself: I'm Joris, a senior front-end engineer with 7 years of professional experience. Throughout my career, I've developed a strong proficiency in modern JavaScript frameworks and libraries, including React, Next.js, and Ember.js.

What truly distinguishes me is my recent involvement in some remarkable projects that revolve around AI integration, particularly in the realm of generative AI, within a few dynamic startup environments. It's no secret that AI integration is a burgeoning trend in our industry, and I've had the privilege of harnessing my front-end development skills in conjunction with AI-driven technologies, thereby gaining invaluable experience in this field.

To provide you with a glimpse of my work, you can explore a couple of examples:

[TryItOnAI](https://tryitonai.com)
[Experiwear](https://experiwear.com)
I'm also more than willing to share portions of the source code or screenshots running locally to give you a firsthand look at my experience.

Having worked in the startup landscape, I fully understand the need for rapid progress and the constraints of limited financial resources. Your project aligns perfectly with my skill set and expertise, and I'm eager to participate in it.

The good news is that I'm currently available and ready to start immediately. To discuss further, please feel free to reach out to me at jorisbaly1@gmail.com.

For your convenience, here's a link to my LinkedIn profile: [Joris Baly on LinkedIn](https://linkedin.com/in/joris-baly).

I'm looking forward to the possibility of working together and eagerly await your response.

Best regards,

Joris

**Comment by SuperButterfly** [comment]
Hi Nice to meet you.
I am s senior front end developer with over 10 years of experience.
I have rich experience in React, Next.js and Vue.js.

These are my previous works:

https://www.craftymail.app

This is a web-based, no-code email builder that allows users to create
beautiful email templates by utilizing a simple drag-and-drop functionality
to rearrange blocks of images, text, and other tools. It was built with
Next.js

https://mydentalgig.com (React.js + Node.js + Next.js + Postgres)
- Created responsive components from the sketch designs.
- Created unit tests for React components using Jest
- integrated REST APIs using Redux-Saga

https://beatswitch.com.
On this project, I used Next.js framework to render React on the
server-side with Node.js backend to improve SEO. I used typescript and
immutable.js to manage complex state.



This is my linkedin profile:  linkedin.com/in/edward-murray-3419aa217
This is my github profile: github.com/superbutterfly

**Comment by Khaan25** [comment]
Hey Hey, Sr. Frontend Dev here with 4 years of experience... From **Pakistan**!
I've expertise in Web Dev and you can find more insights here: https://zzia.vercel.app/about
You will find a link to my resume and the tech stack I know and I've worked on.

You can view my projects here: https://zzia.vercel.app/projects
Each Project has a deep breakdown, tech stack used, end goal, end product, screenshots, live link

You can view my portfolio here: https://zzia.vercel.app
I've mentioned the services I can provide you to boost your sales by improving your website.

LinkedIn: https://www.linkedin.com/in/ziawebdev
GitHub: https://github.com/Khaan25

Best Regards,
Zia Ur.


![og](https://github.com/vercel/next.js/assets/72739794/57e04172-243e-4462-96c7-50cc4b903bd1)
